CMakeLists.txt

```diff
@@ -585,7 +585,9 @@ else()
   check_cxx_symbol_exists(SO_BUSY_POLL sys/socket.h ZMQ_HAVE_BUSY_POLL)
 endif()
 
-if(NOT MINGW)
+if(MINGW)
```

Comment:
Should probably be done in the Windows block above, where the other checks for the same libs are being done?

Reply:
Thanks! Reworked.